ux/designs/integration_details/integration_details_page.md

```diff
@@ -0,0 +1,115 @@
+# Integration Details Page
+
+The designs covers the following user interactions:
+- Integration Details Page basic layout
+  - Expandable row for integration history (versions)
+  - Kebab menu
+  - Integration visualization (normal vs. many steps/connections)
+- Creating/Replacing integration draft and publish a version from integration history
+  - Status indicators at row level
+  - Having notifications in place for publish action
+  - Modal for confirming user actions
+
+## Overview
+
+### Integration Details Page
+
+#### No Draft  
+
+![no draft section](img/integration_details_page_rework_11.16_no_draft.png)
+
+1. If there’s no draft for an integration. The draft section would not be shown on this page.
+
+2. Users would be able to “Create Draft” from the current version.
+
+#### Basic layout
+
+![basic layout](img/integration_details_page_rework_11.16_genericlayout.png)
+
+1. Moved status label to the top section right next to Integration Name. Using “check mark” icon to indicate this integration is active.
+
+2. Moving Draft out of the integration history. Users would be able to “Publish” and “Edit” the draft.
+
+3. Added row indicator (green bar) and checkmark icon to indicate active and running version. For active version, users would only be able to perform “Replace Draft” with the active version.
+
+![many steps and connections](img/integration_details_page_rework_11.16_many_stepsconnections.png)
+
+1. Showing an integration that has many connections and steps.
+
+#### Expanding a row
+
+![expandable row and scrolling](img/integration_details_page_rework_11.16_expand_scroll.png)
+
+1. When expanding a row item, show more details about that version.
+
+  - Start Date/Time
+  - Las Run
+  - Last Modified: user name and timestamp
+  - Modification Notes: works like commit message or note to self
+
+2. Users are able to scroll on this page. When scrolling, the whole page scrolls.
+
+
+### Interacting with the page
+
+#### Kebab menu
+
+![kebab menu](img/integration_details_page_rework_11.16_kebab.png)
+
+1. Breadcrumb & toolbar level kebab, users are able to:
+
+  - Stop Integration
+  - Replace Draft
```

Comment:
With what would the Draft be replaced (as this is a menu without revision context) ?

Maybe a "Remove Draft" would be better at this point.

Reply:
Replace Draft in this context means "replace the existing draft with a copy of the running integration"